apps/protocol/lib/lexical/protocol/requests.ex

```diff
@@ -33,6 +33,12 @@ defmodule Lexical.Protocol.Requests do
     defrequest "textDocument/definition", Types.Definition.Params
   end
 
+  defmodule CreateWorkDoneProgress do
```

Comment:
As `CreateWorkDoneProgress` is a request, I think keeping the name consistent with the title is more appropriate.

Reply:
https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#window_workDoneProgress_create

Reply:
ok.